python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -46,641 +75,2962 @@ def __init__(self, regionname=None, **kwargs):
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def ResumePickPlace(self, timeout=10, **kwargs):
+    def ResumePickPlace(self, timeout=10, dynamicEnvironmentState=None, debuglevel=None, numTargetsToPlace=None, enablerobotbridge=None, **kwargs):
+        # type: (float, Optional[types.ResumePickPlaceParametersDynamicEnvironmentState], Optional[int], Optional[int], Optional[bool], Optional[Any]) -> Optional[Any]
         """
         Args:
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
+            numTargetsToPlace: The number of targets that are left to place (Default: None)
+            enablerobotbridge: If True, enables robot bridge (Default: None)
         """
         taskparameters = {
             'command': 'ResumePickPlace',
-        }
+        }  # type: dict[str, Any]
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
+        if numTargetsToPlace is not None:
+            taskparameters['numTargetsToPlace'] = numTargetsToPlace
+        if enablerobotbridge is not None:
+            taskparameters['enablerobotbridge'] = enablerobotbridge
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def SendStateTrigger(self, stateTrigger, timeout=10, fireandforget=False, **kwargs):
+    def SendStateTrigger(self, stateTrigger, timeout=10, fireandforget=False, dynamicEnvironmentState=None, debuglevel=None, **kwargs):
+        # type: (str, float, bool, Optional[types.SendStateTriggerParametersDynamicEnvironmentState], Optional[int], Optional[Any]) -> Optional[Any]
         """
         Args:
-            stateTrigger (str): a string that represents a unique trigger
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
-            fireandforget (bool, optional): If True, does not wait for the command to finish and returns immediately. The command remains queued on the server. (Default: False)
+            stateTrigger: a string that represents a unique trigger
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            fireandforget: If True, does not wait for th
```

Comment:
Not sure why this wasn't deprecated earlier, but may be this is still an opportunity. 


```
python/mujinplanningclient/packingplanningclient.py
575:    def ValidatePackFormationResultList
```

Reply:
Not very polite, but our API is still unstable. Let's note it explicitly in the changelog.